core/src/main/scala/akka/persistence/cassandra/query/EventsByTagPublisher.scala

```diff
@@ -260,6 +261,21 @@ private[query] class EventsByTagPublisher(
         stopIfDone()
       else
         buf :+= env
+
+      if (buf.size > maxBufferSize) {
+        log.warning(
+          s"${backtrackingLog}, buffer size exceeded. Shouldn't happen, please report issue. " +
+          s"Got event for tag [{}] from pid [{}] with seqNr [{}] and offset [{}], buffered [${buf.size}]",
+          tag, env.persistentRepr.persistenceId, env.persistentRepr.sequenceNr, formatOffset(env.offset)
+        )
+      }
+
+      // This is useful when debugging but too much logging to have enabled by default
+      // if (log.isDebugEnabled)
+      //   log.debug(
+      //     s"${backtrackingLog}got event for tag [{}] from pid [{}] with seqNr [{}] and offset [{}], buffered [${buf.size}]",
+      //     tag, env.persistentRepr.persistenceId, env.persistentRepr.sequenceNr, formatOffset(env.offset))
```

Comment:
Some times I wish we had `trace` as a log level as well, for those few really intense debug logging things.